xapian-applications/omega/handler_tesseract.cc

```diff
@@ -63,6 +63,12 @@ extract(const string& filename,
     }
     // Open Image
     Pix* image = pixRead(filename.c_str());
+
+    if (!image) {
+	error = "Tesseract Error: Error while openning the image";
```

Comment:
Typo: "opening"

Sorry about the English language...

Reply:
I am going to change it.